vmdb/Gemfile

```diff
@@ -42,7 +42,13 @@ gem "acts_as_tree",                   "~>2.1.0"  # acts_as_tree needs to be requ
 # https://github.com/jeremyevans/ruby-american_date
 gem "american_date"
 gem "default_value_for",              "~>2.0.3"
-gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+
+if RUBY_PLATFORM == 'java'
+  gem "puma"
+else
+  gem "thin",                           "~>1.3.1"  # Used by rails server through rack
+end
```

Comment:
fortunately, the ui/webserice worker invocation is concentrated in one place so if we need to divorce from launching it through rails server, [here's](https://github.com/ManageIQ/manageiq/blob/03c3c832732fe08f521a306ef76ee44a9e2a4ad3/vmdb/app/models/mixins/miq_web_server_worker_mixin.rb#L23-L53) where we'd do it.